metrictank-sample.ini

```diff
@@ -188,10 +188,10 @@ net-max-open-requests = 100
 [cluster]
 # The primary node writes data to cassandra. There should only be 1 primary node per shardGroup.
 primary-node = true
-# http/s addresses of other nodes, comma separated. use this if you shard your data and want to query other instances
+# the TCP/UDP address to listen on for the gossip protocol.
+bind-addr = 0.0.0.0:7946
+# TCP addresses of other nodes, comma separated. use this if you shard your data and want to query other instances
```

Comment:
seems like you can specify`host:port` peers, but also `host` peers which will be polled on the same port as the one we're binding on. let's clarify that